edgelet/iotedge/src/check/checks/container_connect_upstream.rs

```diff
@@ -139,16 +139,18 @@ impl ContainerConnectUpstream {
             &port,
         ]);
 
-        let proxy = settings
-            .agent()
-            .env()
-            .get("https_proxy")
-            .map(std::string::String::as_str);
-        self.proxy = proxy.map(ToOwned::to_owned);
-        if let Some(proxy) = proxy {
-            args.extend(&["--proxy", proxy]);
+        if port == 443.to_string() {
+            let proxy = settings
+                .agent()
+                .env()
+                .get("https_proxy")
+                .map(std::string::String::as_str);
+            self.proxy = proxy.map(ToOwned::to_owned);
+            if let Some(proxy) = proxy {
+                args.extend(&["--proxy", proxy]);
+            }
         }
-
+        
```

Comment:
remove trailing whitespace

Reply:
make sure to run `cargo fmt --all` and `cargo clippy --tests` before checking in.

Reply:
Yes I realized I didn't fmt the last bit. Since I run test on VM I don't always have it automated